File: back/KaemeWebApi/Dominio/Models/PedidoModels/PedidoInformation.cs
namespace Dominio.Models
{
    public partial class PedidoInformation
    {
        public int? Num_Pedido { get; set; }
        public int Cliente_Id { get; set; }
        public string? Cpf { get; set; }
        public int Usuario_Id { get; set; }
        public DateTime Data_Pedido { get; set; }
        public bool Ativo { get; set; }
        public bool Cancelado { get; set; }
        public int Status_Pedido_Id { get; set; }
        public string Funcionario { get; set; }
        public string Cliente { get; set; }
        public string Status_Pedido { get; set; }
        public decimal Valor_Total { get; set; }
        public decimal? Valor_Pedido { get; set; }
        public decimal? Valor_Pago { get; set; }
        public DateTime? Data_Pagamento { get; set; }
        public string? Tipo_Pagamento { get; set; }
        public int? Parcelas { get; set; }
        public bool Pago { get; set; }
        public int? Peca_Id { get; set; }
        public string? Peca_Codigo { get; set; }
        public string? Tipo_Peca { get; set; }
        public int? Quantidade { get; set; }
        public decimal? Valor_Peca { get; set; }

    }
}

File: front/kaemeweb/src/app/modules/shared/models/PedidoModels/pedido-information.ts
export interface PedidoInformation {
  num_Pedido: number | null;
  cliente_Id: number;
  usuario_Id: number;
  data_Pedido: Date;
  ativo: boolean;
  cancelado: boolean;
  status_Pedido_Id: number;
  funcionario: string;
  cliente: string;
  status_Pedido: string;
  valor_Total: number;
  valor_Pedido: number | null;
  valor_Pago: number | null;
  data_Pagamento: Date | null;
  tipo_Pagamento: string | null;
  parcelas: number | null;
  pago: boolean;
  peca_Id: number | null;
  peca_Codigo: string | null;
  quantidade: number | null;
  valor_Peca: number | null;
}
